usersimcrs/dialogue_management/dialogue_state.py

```diff
@@ -0,0 +1,31 @@
+"""Representation of the dialogue state.
+
+The dialogue state includes the turn count, a list of dialogue acts per turn for
```

Comment:
(Not a comment for this specific file, but in general) Do we state somewhere how we define turns? I.e., is each utterance by a participant a turn or is an exchange of utterances a turn?

Reply:
The docstring is updated to explicitly describe what is a turn.